lib/IRGen/GenDecl.cpp

```diff
@@ -1881,6 +1881,9 @@ llvm::Constant *IRGenModule::emitProtocolConformances() {
   case llvm::Triple::ELF:
     sectionName = ".swift2_protocol_conformances";
     break;
+  case llvm::Triple::COFF:  // for cygwin
```

Comment:
Please drop the comment.  This code will automatically get activated in more cases than just Cygwin, so it is misleading.